d20/Actions/__init__.py

```diff
@@ -1,39 +1,42 @@
 import sys
 import os
-from typing import Dict
+from d20.Manual.Config import Configuration
 
 from d20.Manual.Options import Arguments
 from d20.Manual.Logger import logging
 
+from typing import Callable, Dict, Type, Optional, List
```

Comment:
Fix the ordering of imports to match PEP8 guidelines, which specify  they should be in the order of standard library ones first, then third party, then local.
[PEP8 import style guidelines](https://www.python.org/dev/peps/pep-0008/#imports)